test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/EditRulingDetailsViewSpec.scala

```diff
@@ -18,15 +18,19 @@ package uk.gov.hmrc.tariffclassificationfrontend.views.partials
 
 import java.time.Instant
 
-import uk.gov.hmrc.tariffclassificationfrontend.forms.{DecisionForm, DecisionFormData}
+import uk.gov.hmrc.tariffclassificationfrontend.forms.{CommodityCodeConstraints, DecisionForm, DecisionFormData}
 import uk.gov.hmrc.tariffclassificationfrontend.models.response.ScanStatus
 import uk.gov.hmrc.tariffclassificationfrontend.models.{CaseStatus, StoredAttachment}
 import uk.gov.hmrc.tariffclassificationfrontend.views.ViewMatchers._
 import uk.gov.hmrc.tariffclassificationfrontend.views.ViewSpec
 import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.ruling_details_edit
 import uk.gov.tariffclassificationfrontend.utils.Cases._
+import org.scalatest.mockito.MockitoSugar
 
-class EditRulingDetailsViewSpec extends ViewSpec {
+class EditRulingDetailsViewSpec extends ViewSpec with MockitoSugar {
+
+  private val commodityCodeConstraints = mock[CommodityCodeConstraints]
+  private val decisionForm = new DecisionForm(commodityCodeConstraints)
```

Comment:
Mock the form rather than the constraint?

Reply:
A bit fiddly to mock the DecisionForm as the test code needs to access parts of the DecisionForm like `form`